hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -343,6 +343,11 @@
   public static final double
       HDDS_DATANODE_STORAGE_UTILIZATION_CRITICAL_THRESHOLD_DEFAULT = 0.95;
 
+  public static final String
+      HDDS_DATANODE_METADATA_CACHE_SIZE = "hdds.datanode.metadata.cache.size";
```

Comment:
I think a better name for this parameter might be:

"hdds.datanode.container.rocksdb.cache.size"

That more clearly says "this is the cache size for the container rocksDBs". What do you think?

Reply:
My initial thought was that *metadata* tells what the cache is used for, which seemed more useful than *rocksdb*, which tells how the cache is implemented. But if this config is being changed, knowing that it is for a RocksDB instance might be useful. Maybe we could include both parts? For example, the existing config *ozone.metastore.rocksdb.statistics* contains both pieces of information.

I think putting *container* in the name makes the config sound like it is for one container, whereas just having *datanode* makes it more apparent that this is the size of one shared cache for the whole DN.

These would give a new name like *hdds.datanode.rocksdb.metadata.cache.size* or *hdds.datanode.metadata.store.rocksdb.cache.size* which are informative but fairly long. Since ozone-default.xml has a description, this may not be necessary.

I don't really have a strong opinion either way. If you think *hdds.datanode.container.rocksdb.cache.size* the best name I am fine with that.

Reply:
How about `hdds.datanode.metadata.rocksdb.cache.size`? I just dropped `store` from your suggestion.

Reply:
Sounds good to me, I'll update the name.